api/envoy/api/v2/cluster/outlier_detection.proto

```diff
@@ -114,4 +114,33 @@ message OutlierDetection {
   // is set to true.
   google.protobuf.UInt32Value enforcing_local_origin_success_rate = 15
       [(validate.rules).uint32.lte = 100];
+
+  // The failure percentage to use when determining failure percentage-based outlier detection. If
+  // the failure percentage of a given host is greater than or equal to this value, it will be
+  // ejected. Defaults to 85.
```

Comment:
Is this a reasonable default? I picked this entirely arbitrarily, so I have no attachment to it.

Reply:
So, just to clarify here - alright to leave it as is now and tag with a `[#next-major-version]` comment?

Reply:
definitely fine to leave as-is.  Bonus points if you file a tech debt issue, and mention it should be tagged with v4 (I don't think you can add tags)

Reply:
+1 on filing a tech debt issue on cleaning up all of these messages, agreed it is a big mess as it has evolved quite a bit trying to not break backwards compat. I can tag it correctly once it is opened.

Reply:
I've filed #8219 for this one.